velox/functions/prestosql/ArrayFunctions.h

```diff
@@ -728,6 +728,38 @@ struct ArrayRemoveNullFunctionString {
   }
 };
 
+template <typename T>
+struct ArrayNGramFunction {
```

Comment:
nit: ArrayNGramFunction -> ArrayNGramsFunction (add an 's' to match function name: ngrams)